src/plots/cartesian/axes.js

```diff
@@ -100,33 +99,20 @@ axes.coerceRef = function(containerIn, containerOut, gd, attr, dflt, extraOption
  * - for other types: coerce them to numbers
  */
 axes.coercePosition = function(containerOut, gd, coerce, axRef, attr, dflt) {
-    var pos,
-        newPos;
+    var ax, pos;
 
     if(axRef === 'paper' || axRef === 'pixel') {
+        ax = { cleanPos: Lib.num };
```

Comment:
don't need to pretend we're making an axis anymore... just `cleanPos = Lib.num` or `cleanPos = ax.cleanPos`